rpcserver.go

```diff
@@ -3368,6 +3369,37 @@ func handleGetNetworkHashPS(s *rpcServer, cmd interface{}, closeChan <-chan stru
 	return hashesPerSec.Int64(), nil
 }
 
+// handleGetNetworkInfo implements the getnetworkinfo command.
+func handleGetNetworkInfo(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (interface{}, error) {
+	networks := cfg.generateNetworkInfo()
+
+	lAddrs := s.server.addrManager.FetchLocalAddresses()
+	localAddrs := make([]dcrjson.LocalAddressesResult, len(lAddrs))
+	for _, entry := range lAddrs {
+		addr := dcrjson.LocalAddressesResult{
+			Address: entry.Address,
+			Port:    entry.Port,
+			Score:   entry.Score,
+		}
+		localAddrs = append(localAddrs, addr)
+	}
+
+	info := dcrjson.GetNetworkInfoResult{
+		Version: int32(1000000*version.Major + 10000*version.Minor +
+			100*version.Patch),
+		SubVersion:      userAgentVersion,
+		ProtocolVersion: int32(maxProtocolVersion),
+		TimeOffset:      int64(s.server.timeSource.Offset().Seconds()),
+		Connections:     s.server.ConnectedCount(),
+		RelayFee:        cfg.minRelayTxFee.ToCoin(),
+		Networks:        networks,
+		LocalAddresses:  localAddrs,
+		LocalServices:   s.server.services.String(),
```

Comment:
This should be a hexadecimal value. This currently will result in the `fmt.Stringer` being called which produces the named flags.

e.g.

`"localservices": "SFNodeNetwork|SFNodeCF"`